### ISSUE TITLE
Suppress code in ipython console

### ISSUE THREAD

**Issue by rathsandeep** [pancho111203/vscode-ipython#2]
This is a great extension. I really appreciate the fact that after the code is run it returns back an ipython prompt. Is there a way to suppress the code in the console? Right now after multiple runs, there is a lot of code in the console history, and searching something a couple of iterations back is quite difficult.

**Comment by jrwrigh** [comment]
Yeah, maybe something like `%run -i {filename}` would work better than `%load {filename}`. I also don't necessarily want to see the code itself in the console as much as I really just want to run it.
